packages/block-library/src/comments-query-loop/order-controls.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+import { __, _x } from '@wordpress/i18n';
+
+export default function CommentsQueryOrderControls( { value, onChange } ) {
+	return (
+		<ToggleGroupControl
```

Comment:
It doesn't seem that the `ToggleGroupControl` is the right one for this control.. I think a combo box control is a better fit here and will accommodate for more options, if they need to be added.

Reply:
Thanks for the review!!

 It seems that order options only accepts two values here, a long time ago for now. This is what we have when we check the documentation about the comments arguments on that function.
<img width="1042" alt="Screenshot 2021-12-13 at 10 29 22" src="https://user-images.githubusercontent.com/37012961/145787090-318d34ad-5b07-4192-be49-0796e8acb89c.png">

Maybe the combo box would be a better option for OrderBy?
<img width="677" alt="Screenshot 2021-12-13 at 10 32 02" src="https://user-images.githubusercontent.com/37012961/145787052-e8a7b438-dc6c-4376-b8c3-914f52fa4b75.png">

Reply:
I think the common case is to combine `order and orderBy` in a single control when we want both, but if we want only a control with two values (asc, desc) here, we might even consider a simple toggle control? --cc @jasmussen

Reply:
Nice PR! Yes, as a rule of thumb, if the text labels wrap inside a segmented control, then it's not the right choice. I would also lean towards a basic toggle in cases where there are only two options. The combobox/dropdown is good for surfacing longer labels and obviously for options that have many options.

Is the toggle right for sort order? I'd say that depends on the phrasing and if it feels right. Something like this might work:

```
[on] Sort from newest to oldest
```

The implication would be that toggling it off would sort from _oldest to newest_. I think I'd personally like this as a good starting point, since it's a fairly simple toggle and good baseline. 

We can always upgrade to the dropdown, which could be something like:

```
Sort order:
[ Newest to oldest
  Oldest to newest ]
```

What do you think?